tests/Core/Tokenizers/PHP/ContextSensitiveKeywordsTest.php

```diff
@@ -236,14 +236,26 @@ public static function dataKeywords()
                 'testMarker'        => '/* testPrivateIsKeyword */',
                 'expectedTokenType' => 'T_PRIVATE',
             ],
+            'private(set): property declaration'     => [
```

Comment:
Nitpick: please let the data provider follow the same order as the tests in the test case file.